File: char_swap.py
# Given a string swap all the char X with char Y.
str = input("Enter the string\n")
res = ""
c1='a'
c2='b'
for i in range(len(str)):
    if(str[i]!=c1):
        res += str[i]
    else:
        res += c2
print(res)


# def replaceCharacter(str, c1, c2):
#     n = len(str)
#     res = ""
#     for i in range(n):
#         if str[i] != c1:
#             res += str[i]
#         else:
#             res += c2
#     return res
 
# # Driver Code
# if __name__ == '__main__':
#     # Given string
#     str = "abacd"
#     c1 = 'a'
#     c2 = 'x'
 
#     # Function call
#     str = replaceCharacter(str, c1, c2)
 
#     # Print the string
#     print(str)

File: Char_Swap.java
import java.util.Scanner;
public class Char_Swap {
    public static void main(String[] args) {
            Scanner sc = new Scanner(System.in);
            String s1 = ("Abaaaba ok get ako");
            String s2 = s1.replace('a', 'b');
            System.out.println(s2);
    }
}
//
//public class Main {
//    static void replaceCharacter(char[] str, char c1, char c2) {
//        int j = 0;
//        int n = str.length;
//        for (int i = 0; i < n; i++) {
//            if (str[i] != c1) {
//                str[j++] = str[i];
//            } else {
//                str[j++] = c2;
//            }
//        }
//    }
//
//    public static void main(String[] args) {
//        // Given string
//        char[] str = "abacd".toCharArray();
//        char c1 = 'a';
//        char c2 = 'x';
//
//        // Function call
//        replaceCharacter(str, c1, c2);
//
//        // Print the string
//        System.out.println(str);
//    }
//}